sdk/openai/azure-ai-openai-assistants/src/main/java/module-info.java

```diff
@@ -4,7 +4,7 @@
 
 module com.azure.ai.openai.assistants {
     requires transitive com.azure.core;
-    requires transitive com.azure.json;
+    requires com.azure.json;
```

Comment:
We should keep this transitive.

Reply:
From the migration guideline, we don't require the transitive.

Reply:
This is not needed as azure-core is already doing this. We should update the migration guide.